app/move/controllers/view.js

```diff
@@ -1,6 +1,19 @@
+const moveService = require('../../../common/services/move')
+
 const getViewLocals = require('./view/view.locals')
 
-module.exports = function view(req, res) {
+module.exports = async function view(req, res) {
+  const { id, status, cancellation_reason: cancellationReason } = req.move
+
+  // TODO: remove this second call when backend returns rebook info as part of move
+  if (status === 'cancelled' && cancellationReason === 'rejected') {
```

Comment:
Could we do this in the new "transformer" for now instead? If it's essentially something that we're going to expect on the move that feels like a nice place to do this patch for now.

Reply:
I disagree. This is a kludge until the backend does the right thing. Moreover, moving it the transformer would mean that we would (unnecessarily) perform this transformation for every rejected single request when listing them.